spec/mocks/MockCompiler.coffee

```diff
@@ -0,0 +1,57 @@
+$ = require('jquery')
+Backbone = require('backbone')
+
+MockCompiler = Backbone.Model.extend(
+  load: (templateName, onCompletion) ->
+    html = @getHTMLForTemplateName(templateName)
+    onCompletion(html)
+
+  appendToContainer: (container, template) ->
+    $(container).html(template)
+
+  getHTMLForTemplateName: (templateName) ->
```

Comment:
It might be easier to pass in a function to apply to the template rather than specifying to use .html. In the future, I might need to use .append or something else.